esp/services/ws_dfu/ws_dfuXRefService.cpp

```diff
@@ -553,6 +553,9 @@ bool CWsDfuXRefEx::onDFUXRefList(IEspContext &context, IEspDFUXRefListRequest &r
 {
     try
     {
+#ifdef _CONTAINERIZED
+        IERRLOG("CONTAINERIZED(CWsDfuXRefEx::onDFUXRefList)");
```

Comment:
Why IERRLOG here but UNIMPLEMENTED below?

Reply:
I tried to use UNIMPLEMENTED when the code would have previously failed, and IERRLOG when it would have silently ignored it.